Source/WebKit/WebProcess/Plugins/PDF/PDFPlugin.mm

```diff
@@ -1276,7 +1276,7 @@ static void dataProviderReleaseInfoCallback(void* info)
 PluginInfo PDFPlugin::pluginInfo()
 {
     PluginInfo info;
-    info.name = builtInPDFPluginName();
+    info.name = "WebKit built-in PDF"_s;
```

Comment:
Given that we don't have a regression test, seems worth adding a comment about why this shouldn't be translated.